Atlantis, KI Praktikum 2025 [Our estimate > 95% complete]

zatakashish opened the pull request.

sebiwtt commented:
- You do merge some “random” files ❌
    - Do we need the sequence_diagram_pong.md and the sprive_visualizer.py script or are they just for your own convenience during dev work?
- Wants to merge into master ❌ → change to dev please :)
- Runs with play.py ✅
- Implements all functions from base env ✅
- Completion: The game looks very nice! When comparing it to the ALE version also found it to be very close! Do the entities on the “bottom” do stuff? From your 95% estimate I would guess so :D. Do you guys know what you need to add for the last 5%? Thanks for the great work!

zatakashish commented:
@sebiwtt 

Thank you for you review.

I changed the target branch to dev ✅

I removed the unwanted files ✅ (sequence diagram and sprite visualizer)

Regarding the 95 % completion: we currently have no outstanding issues, so we’ve reserved a 5 % buffer for any minor cosmetic fixes. From our standpoint, the game is effectively finished.
